editor.planx.uk/src/pages/FlowEditor/components/Team/components/MembersTable.tsx

```diff
@@ -98,14 +101,14 @@ export const MembersTable = ({
                     addUser();
                   }}
                 >
-                  Add a new editor
+                  Add a new member
                 </AddButton>
               </TableCell>
             </TableRow>
           )}
         </Table>
         {showModal && (
-          <EditorUpsertModal
+          <UserUpsertModal
```

Comment:
There's still a distinction between a `user` and `team_members` that I think we want to maintain. This modal adds a `user` record to the db, and then assigns them a role.